docs/dev/table/connectors/jdbc.md

```diff
@@ -153,7 +153,7 @@ Connector Options
       <td>optional</td>
       <td style="word-wrap: break-word;">60s</td>
       <td>Duration</td>
-      <td>Maximum timeout between retries.</td>
+      <td>The timeout should be in second granularity and shouldn't be smaller than 1 second.</td>
```

Comment:
Um....  We don't need to remove the original description, that is the most important description for this option (describe WHAT IS THIS OPTION USED FOR). I mean we just add the additional restriction description.

Reply:
ok，I merged them